scripts/host-display

```diff
@@ -80,6 +81,17 @@ COMMANDS = {
     'status': status
 }
 
+def _get_com_port():
+    """
+        Determine from the current bootloader config which COM port is in use
+    """
+    (_, stdout, _) = doexec([XEN_CMDLINE, "--get-xen", "console"])
+    m = re.match("console=.*(com\d).*", stdout.readline().strip())
+    if m:
+        return m.group(1)
+    else:
+        return "com1"
```

Comment:
If everything goes well this shouldn't be reached, unless there was a previous run with the old host-display script that wiped the com1 entry.

Reply:
Indeed, I felt it a good precaution to have something present just in case however